### DISCUSSION TITLE
communication-ble-client receive: cannot compile :error [SOLVED]

### DISCUSSION THREAD

**Discussion by TangerineDreamer** [pschatzmann/arduino-audio-tools#1157] [Q&A] [answered]
When i want to compile this sketch it shows this error:

"cannot declare variable 'ble' to be of abstract type 'audio_tools::AudioBLEClient'"

It happens also with the sketch "communication-ble-client-send"

**Comment by pschatzmann** [comment, marked as answer]
I committed a small correction to make it compile again

**Comment by TangerineDreamer** [comment]
Compiled fine and working fine too ,but i better like the generator sound in the format: AudioInfo info(22000, 1, 16);  :)
thank you!
